onedal/basic_statistics/tests/test_basic_statistics.py

```diff
@@ -98,3 +106,79 @@ def test_option_weighted(queue, option, dtype):
 
         tol = fp32tol if res.dtype == np.float32 else fp64tol
         assert_allclose(gtr, res, rtol=tol)
+
+    @pytest.mark.parametrize("queue", get_queues())
+    @pytest.mark.parametrize("dtype", [np.float32, np.float64])
+    def test_basic_csr(queue, dtype):
+        from packaging.version import Version
+        from scipy import __version__
+
+        # Support of scipy.sparse.random_array was added in scipy 1.12.0
+        # Deselecting the test for earlier versions
+        if Version(__version__) < Version("1.12.0"):
+            pytest.skip("skipping BasicStatistics CSR test for SciPy < 1.12.0")
+
+        seed = 42
+        s_count, f_count = 5000, 3008
+
+        gen = np.random.default_rng(seed)
+
+        from scipy.sparse import random_array
+
+        data = random_array(
```

Comment:
I am disabling the test only for older versions of scipy. The stable version is 1.13.1, it supports the primitive, and the upcoming releases will do as well.

I don't think it is a good idea to use dense representation - it will consume too much memory and we won't be able to test the dimensions typical for sparse data.